docs/findings/el9.md

```diff
@@ -330,5 +335,31 @@ These are files that _need_ to set to mode `4755` &ndash; permissions that are b
 
 Any files other than `/lib/polkit-1/polkit-agent-helper-1` and `/usr/lib/polkit-1/polkit-agent-helper-1` should be treated as valid findings and remediated.
 
+# The OS must require authentication to access single-user mode
+
+**Invalid Finding:**
+
+Some scanners may call out this setting as incorrect, even if correctly-set.
+
+The value as described in STIG v2r4 includes guidance suggesting setting a value of:
+
+```bash
+/usr/lib/systemd/systemd-sulogin-shell rescue
+```
+
+However, the Ansible content that comes with the `oscap` content from the Compliance As Code project implements setting a value of:
+
+```bash
+/bin/sh -c "/sbin/sulogin ; /usr/bin/systemctl --fail --no-block default"
+```
+
+Regardless of method chosen, some scanners will still flag the service-configuration as non-compliant. STIG guidance indicate that so long as:
+
+```bash
+grep sulogin /usr/lib/systemd/system/rescue.service
+```
+
+Returns either of the above, two values that the scan-target is adequately configured for this finding.
```

Comment:
Just a nit, but should that comma be after "two values"? My brain is hitting a halt and core dump scenario trying to parse as is...

```suggestion
Returns either of the above two values, that the scan-target is adequately configured for this finding.
```

Reply:
The "above" and "two" are both modifiers (adjectives) for "values" (noun). Grammar rules state that adjectives in series should be comma-separated (see documents like [this one](https://prowritingaid.com/comma-between-two-adjectives)).